docs/user-guide/global-configurations/chart-repo.md

```diff
@@ -0,0 +1,56 @@
+# Chart Repository
+
+The Global configuration provides a feature of `chart repository` in which you can add your chart repository here.
+
+## Add Chart Repository
+
+Select the Chart Repository section of global configuration and click on `Add Repository` to add new chart, which provide three inputs as below.
+
+1. Name
+2. URL
+3. Authentication type
+
+![](../../.gitbook/assets/gc-add-chart.png)
+
+### 1. Name
+
+Provide a `Name` to your Chart Repository. It will be added as one option in the Chart Repository drop-down inside the Chart Repository section.
+
+### 2. URL
+
+Provide the `URL` of your Version Controller. **For example**- github.com for Github, [https://gitlab.com](https://gitlab.com) for GitLab, etc.
+
+### 3. Authentication type
+
+Here you have to provide the type of Authentication required by your version controller. We are supporting three types of authentications, you can choose any one from them.
+
+
+* **Anonymous**
+
+If you select `Anonymous` then you do not have to provide any username, password, and authentication token. Just click on `Save` to save your chart repository details.
+
+![](../../.gitbook/assets/gc-chart-configure-anonymous.png)
+
+* **Password/Auth token**
+
+If you select Password/Auth token then you have to provide the `Access token` for the authentication of your version controller account inside the Access token box. Click on `Save` to save your chart repository details.
+
+![](../../.gitbook/assets/gc-chart-config-password.png)
+
+* **User Auth**
+
+If you choose `User Auth` then you have to provide the `Username` and `Password` of your version controller account. Click on `Save` to save your chart repository details.
+
+![](../../.gitbook/assets/gc-chart-configure-user.png)
+
+## Update Chart Repository
+
+You can update your saved chart repository settings at any point in time. Click on the chart repository which you want to update. Make changes and click on \`Update to save you changes.
```

Comment:
Update to save your changes has open tick but not closing tick, instead of  '\`Update -> '`Update`'